concert/app/src/functions/on_event_stage_changed/index.py

```diff
@@ -19,4 +35,36 @@ def handler(event, context):
         },
     )
 
+    song_list_update_response = update_song_list(message)
+
     return {"statusCode": 204}
+
+
+def update_song_list(message):
+    response = None
+
+    data = message["data"]
+    display_name = data.get(ATTR_DISPLAY_NAME)
+    start_time = data.get(ATTR_START_TIME)
+    end_time = data.get(ATTR_END_TIME)
+    if display_name and start_time and end_time:
```

Comment:
What about if not?

Reply:
Intent here is to do nothing if not. My understanding is that some `EVENT_STAGE_CHANGED` events -- e.g. mental imagery, intermission -- are not expected to contain any song list information, so for such events, just skip the update.

Reply:
Wouldn't this be a more explicit way of expressing that idea?
```suggestion
    if not (display_name and not start_time and end_time):
        return
    # Continue with function login
```

Early returns are your friend.

Reply:
Done.